<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zadanie6_part1_var1</class>
 <widget class="QWidget" name="Zadanie6_part1_var1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Задание 6.*
 Сгруппируйте синонимы.</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_12">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>420</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_5">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>210</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>determined</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_10">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>360</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_8">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>300</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_answer">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>540</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_15">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_11">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>390</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>usually</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_1">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>90</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_14">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>insulator</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_14">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_4">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>180</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_3">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>150</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>figures</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_4">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>limited</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_13">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>measure</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_9">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>330</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_1">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>including</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_8">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>flowing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_7">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>relationship</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>decrease</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_2">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>120</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>components</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_14">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>480</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_15">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>510</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>advantage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_6">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>240</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_7">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>270</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>wire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_13">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_13">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>450</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_15">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>510</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_2">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>120</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_5">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>210</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_3">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>150</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_6">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>using</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>permit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_answer_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_16">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>360</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>electrical</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_17">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>circuit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_18">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>330</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>often</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_19">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>510</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>proportional</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_20">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>collide</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_21">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>heat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_22">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>390</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>length</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_23">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>commonly</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_24">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>restrict</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_25">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>480</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>inversely</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_26">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>equation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_27">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>420</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>offer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_28">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>450</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>high</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_29">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>containing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word_30">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>electric</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_timer">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>581</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../img/for_tests.jpg</pixmap>
   </property>
  </widget>
  <zorder>label_img</zorder>
  <zorder>label_title</zorder>
  <zorder>lineEdit_answer_12</zorder>
  <zorder>label_number_11</zorder>
  <zorder>label_word_5</zorder>
  <zorder>lineEdit_answer_10</zorder>
  <zorder>label_number_8</zorder>
  <zorder>lineEdit_answer_3</zorder>
  <zorder>pushButton_answer</zorder>
  <zorder>label_number_15</zorder>
  <zorder>lineEdit_answer_11</zorder>
  <zorder>label_number_10</zorder>
  <zorder>label_word_9</zorder>
  <zorder>label_number_1</zorder>
  <zorder>lineEdit_answer_6</zorder>
  <zorder>label_number_12</zorder>
  <zorder>label_word_14</zorder>
  <zorder>lineEdit_answer_8</zorder>
  <zorder>label_number_14</zorder>
  <zorder>label_number_4</zorder>
  <zorder>label_word_3</zorder>
  <zorder>label_word_4</zorder>
  <zorder>label_word_13</zorder>
  <zorder>lineEdit_answer_9</zorder>
  <zorder>label_word_1</zorder>
  <zorder>label_number_9</zorder>
  <zorder>label_word_8</zorder>
  <zorder>label_word_7</zorder>
  <zorder>label_word_10</zorder>
  <zorder>label_word_2</zorder>
  <zorder>lineEdit_answer_4</zorder>
  <zorder>lineEdit_answer_14</zorder>
  <zorder>lineEdit_answer_2</zorder>
  <zorder>label_word_15</zorder>
  <zorder>label_number_6</zorder>
  <zorder>lineEdit_answer_7</zorder>
  <zorder>label_number_7</zorder>
  <zorder>label_word_12</zorder>
  <zorder>label_number_13</zorder>
  <zorder>lineEdit_answer_1</zorder>
  <zorder>lineEdit_answer_13</zorder>
  <zorder>lineEdit_answer_15</zorder>
  <zorder>label_number_2</zorder>
  <zorder>label_number_5</zorder>
  <zorder>label_number_3</zorder>
  <zorder>label_word_6</zorder>
  <zorder>label_word_11</zorder>
  <zorder>lineEdit_answer_5</zorder>
  <zorder>label_word_16</zorder>
  <zorder>label_word_17</zorder>
  <zorder>label_word_18</zorder>
  <zorder>label_word_19</zorder>
  <zorder>label_word_20</zorder>
  <zorder>label_word_21</zorder>
  <zorder>label_word_22</zorder>
  <zorder>label_word_23</zorder>
  <zorder>label_word_24</zorder>
  <zorder>label_word_25</zorder>
  <zorder>label_word_26</zorder>
  <zorder>label_word_27</zorder>
  <zorder>label_word_28</zorder>
  <zorder>label_word_29</zorder>
  <zorder>label_word_30</zorder>
  <zorder>label_timer</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
